.travis.yml

```diff
@@ -1,8 +1,15 @@
 language: php
 
 php:
+  - nightly
   - 5.6
   - 5.5
   - 5.4
   - 5.3
   - hhvm
+  - hhvm-nightly
+
+matrix:
```

Comment:
As I said to you in another place I think fast_finish: true should be added here.